exchange/exchange.go

```diff
@@ -322,6 +325,10 @@ func (e *exchange) getAllBids(ctx context.Context, cleanRequests map[openrtb_ext
 			// Structure to record extra tracking data generated during bidding
 			ae := new(seatResponseExtra)
 			ae.ResponseTimeMillis = int(elapsed / time.Millisecond)
+			if request.Test == 1 && bids != nil {
```

Comment:
Consider removing the `request.Test` check here to always capture the information (there's no overhead for doing so) and letting downstream code decide what to do with it (as it already does with repeating this check).

Reply:
Done!